Makefile

```diff
@@ -134,17 +134,17 @@ test-php-phpstan: vendor-bin/phpstan/vendor
 
 .PHONY: test-acceptance-api
 test-acceptance-api:       ## Run API acceptance tests
-test-acceptance-api: vendor/bin/phpunit
+test-acceptance-api:
 	../../tests/acceptance/run.sh --remote --type api
 
 .PHONY: test-acceptance-cli
 test-acceptance-cli:       ## Run CLI acceptance tests
-test-acceptance-cli: vendor/bin/phpunit
+test-acceptance-cli:
 	../../tests/acceptance/run.sh --remote --type cli
 
 .PHONY: test-acceptance-webui
 test-acceptance-webui:     ## Run webUI acceptance tests
-test-acceptance-webui: vendor/bin/phpunit
+test-acceptance-webui:
```

Comment:
After removing the dependency, new/updated packages are not installed, right? So, tests might fail locally if they depend on external packages?

Currently, though the app does not have any `dev` dependencies.

Reply:
It won't do whatever it did to try and "make" ``vendor/bin/phpunit`` - and actually we do not care about that. But yes, if the app had some ordinary "require" dependencies in ``composer.json`` then I suppose every ``make test*`` target should end up doing ``composer install`` to make those dependencies get loaded. So we do need to think about it in other apps, and put some sensible dependency.

In this case, ``composer install`` in the app will get the ``vendor-bin`` stuff, which is coding-standard, phan, phpstan. And we do not need any of those to run acceptance tests.